<!--
 * This file is part of Blind Camaleon.
 *
 * Blind Camaleon is free software: you can redistribute it and/or modify
 * it under the terms of the GNU Lesser General Public License as published by
 * the Free Software Foundation, either version 3 of the License, or
 * (at your option) any later version.
 *
 * Blind Camaleon is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 * GNU Lesser General Public License for more details.
 *
 * You should have received a copy of the GNU Lesser General Public License
 * along with Blind Camaleon.  If not, see <http://www.gnu.org/licenses/lgpl-3.0.txt>.
-->

<ui version="4.0" >
 <class>IdealFilterDialog</class>
 <widget class="QDialog" name="IdealFilterDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Ideal Filter</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QScrollArea" native="1" name="scrollArea" >
     <widget class="IdealFilteredImage" name="imageLabel" >
      <property name="geometry" >
       <rect>
        <x>14</x>
        <y>14</y>
        <width>503</width>
        <height>389</height>
       </rect>
      </property>
      <property name="text" >
       <string/>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QCheckBox" name="showGuideLines" >
       <property name="text" >
        <string>Show guide lines</string>
       </property>
       <property name="checked" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IdealFilteredImage</class>
   <extends>QLabel</extends>
   <header>idealfilteredimage.h</header>
  </customwidget>
  <customwidget>
   <class>QScrollArea</class>
   <extends>QWidget</extends>
   <header location="global" >QScrollArea</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
